jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/ServletApiRequest.java

```diff
@@ -187,6 +187,8 @@ public String getRequestId()
     @Override
     public String getProtocolRequestId()
     {
+        if (getRequest().getConnectionMetaData().getHttpVersion().getVersion() <= HttpVersion.HTTP_1_0.getVersion())
```

Comment:
This should be `<= HttpVersion.HTTP_2_0.getVersion()` as specified in this method's javadoc.

Reply:
I've switched to only return protocol IDs for protocols known to have IDs